app/models/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,11 @@
+module CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def remote_delete
+    raw_delete
+  end
+
+  def raw_delete
```

Comment:
ok, sorry, only slowly getting there.

We have `Vm#raw_destroy` and `Vm#vm_destroy` or `AuthKeyPair#raw_delete_key_pair` and `AuthKeyPair#delete_key_pair`. So unfortunately there is no consistent naming. `remote_delete` seems to introduce a new naming.

I'd go with `raw_delete_cloud_object_store_container` and `delete_cloud_object_store_container`, but 🤷‍♂️ 
@agrare wdyt ?